### DISCUSSION TITLE
Issue / Feature request: cannot resize files changed sidebar

### DISCUSSION THREAD

**Discussion by james-hill-kc** [community/community#86725] [Pull Requests]
### Select Topic Area

Bug

### Body

After discovering that there is apparently no public issue tracker for github itself (which I find surprising), I raised this as an issue with the support team, but was redirected to this forum (which I'd already attempted to search) to raise it here instead.

Product: GitHub

Description

When using the "files changed" view in a github repository pull request, the sidebar is too narrow by default, and truncates the names of files in even relatively shallow directory hierarchies. There is no obvious ability to resize the sidebar. I would expect to be able to resize the sidebar by dragging it (a feature which is commonly implemented in many applications, for example, in Visual Studio code).

Reproduction Steps

1. Visit any github repository which has pull requests
2. Click on the "pull requests" tab to view the list of pull requests
3. Click on a pull request list item to open the pull request view
4. Click on the "files changed" tab to open the files changed view
5. Observe the file tree sidebar

Fault Type: Permanent - Occurring repeatably

Blocking? No

Last working date: 12 January

Logs: N/A

Similar to:
https://github.com/orgs/community/discussions/3984

Apologies if this duplicates an existing thread/topic, I searched to the best of my ability, and couldn't find one.

Thanks!

**Comment by gguillen** [comment]
+1.  Everyone on my team hates that you cant expand the file tree.

**Comment by taylorplewe** [comment]
+1, and it would also be nice to hide the sidebar entirely

**Reply by AntunBartonicek** [reply]
You can do that now by clicking Show/Hide File Tree but I really miss the ability to resize the sidebar....
![image](https://github.com/user-attachments/assets/5b557cc5-b7e3-4ae8-8d4e-7906dbe26ef8)

**Reply by taylorplewe** [reply]
Not sure how I didn't see that 😅

**Comment by andrewlehmann** [comment]
+1, this is a feature that I used heavily in Bitbucket 5+ years ago.

**Comment by ngoc-dao-cultureamp** [comment]
Please make the sidebar resizable, so that it's easier to review files in pull requests 🙏

**Comment by mrrts** [comment]
Would be great if it would also show the full filename on hover in case you don't want to resize

**Comment by md-mcl** [comment]
I found this workaround, but definitely would be better to make this resizable by default.
https://chromewebstore.google.com/detail/github-file-tree-resizer/fhlpkpfgceoppigefcnejecfaebbflnp?pli=1

**Reply by AntunBartonicek** [reply]
Yes, we can help ourselves by using various extensions but that isn't the point :)
I'm personally using [Tampermonkey ](https://microsoftedge.microsoft.com/addons/detail/tampermonkey/iikmkjmpaadaobahmlepeloendndfphd?refid=bingshortanswersdownload) and a custom script that allows me to:
1. reverse sort for commits
2. reverse sort conversations (to see most recent stuff on top)
3. resize the sidebar (by entering new width)
4. add tooltip to all the files in file tree
Example:
![image](https://github.com/user-attachments/assets/22cde7cc-62e8-4cb5-b8de-c41868ccae9c)

**Comment by stephendeweyEQ** [comment]
I found  this thread using Perplexity, as it's a hard thing to search for. Yes, this would be a big improvement as even relatively short file names can get cut off. If you just let us drag the column to widen it, it would be a big improvement.

**Comment by ChristiaanDuck** [comment]
I can't believe this hasn't been addressed yet! It's infuriating! Almost enough to make me switch to Azure Repos.

**Comment by SFilinsky** [comment]
+1, additionally please add tooltip when hovering file names, so it's possible to understand what file it is without opening it. 

<img width="326" alt="Screenshot 2025-06-25 at 17 00 45" src="https://github.com/user-attachments/assets/f20fdd0b-085d-4969-bb24-1836eb2cd2da" />

**Comment by danielclarkewisetechglobal** [comment]
Can we place have this fixed?  
My enterprise moved from DevOps to Github recently and the UI of the pull requests is driving me crazy (this being one such broken or infuriating features)

**Reply by AntunBartonicek** [reply]
It seems new UI is rolling out so if you try new experience you will be pleasantly surprised :)
<img width="308" height="61" alt="image" src="https://github.com/user-attachments/assets/c7a29b3d-0ba1-4578-80c6-b98184bb7f4d" />
It has drag to resize among other things...
https://github.com/orgs/community/discussions/163932

**Reply by danielclarkewisetechglobal** [reply]
That's great.  Thank you.
